proxy/gateway/java/gateway-remote-module-sdk/src/main/java/com/microsoft/azure/gateway/remote/BrokerProxy.java

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
I removed sdk, but keep gateway-remote-module because nanaomsg-binding is going to be added.